jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -12,21 +12,23 @@ import { sleepMsec } from './extensions/common/misc';
 import { activate as activatecorrelograms } from './extensions/correlograms/correlograms';
 import { activate as activateelectrodegeometry } from './extensions/electrodegeometry/electrodegeometry';
 import { ExtensionContext, HitherContext, HitherJob, HitherJobOpts, Recording, RecordingViewPlugin, Sorting, SortingUnitMetricPlugin, SortingUnitViewPlugin, SortingViewPlugin } from './extensions/extensionInterface';
+import { activate as activatetimeseries } from './extensions/timeseries/timeseries';
```

Comment:
activate the timeseries extension. In the future this section will be auto-generated from a template, similar to what is done for the web gui.